YunoHost/yunohost
# Add default domain to API response

## The problem

The web admin interface needs to know the default domain to select it in the drop down lists. (app install).

https://github.com/YunoHost/issues/issues/1633

...

## Solution

Add a `default` value that contains the index of the default domain in the list.

...

## PR Status
Needs another PR on yunohost-admin repo.
...

## How to test

Just try to install a new app, the domain selection list should be alphabetically sorted and the main domain should be selected.
...
